DatadogSessionReplay/Sources/Recorder/RecordingCoordinator.swift

```diff
@@ -115,46 +119,71 @@ internal class RecordingCoordinator {
         srContextPublisher.setHasReplay(hasReplay)
     }
 
+    private var lastFrameTimestamp: Date?
+    private var shouldSkipFrame: Bool {
+        return dateProvider.now.timeIntervalSince(lastFrameTimestamp ?? .distantPast) < Constants.throttingRate
+    }
```

Comment:
/suggestion We could have a `Debouncer` object that could take care of this, similar to the future `TimeBank`, WDYT?

Reply:
Totally - but for now I'd suggest merging this as is - and refactor once we have timebank in place. It will shed some more slight on our approach of unit testing this